client/object_delete.go

```diff
@@ -46,23 +46,6 @@ func (x *PrmObjectDelete) WithinSession(t session.Object) {
 	x.meta.SetSessionToken(&tv2)
 }
 
-// GetSession returns session object.
-//
-// Returns ErrNoSession err if session wasn't set.
-func (x PrmObjectDelete) GetSession() (*session.Object, error) {
```

Comment:
We're an SDK here, it might still be useful.

Reply:
Returned this function